src/azul/indexer/document.py

```diff
@@ -346,7 +346,8 @@ class Document(Generic[C]):
 
     coordinates: C
     version_type: VersionType = field(init=False)
-    version: Optional[int]
+    # version is a tuple composed of the sequence number and primary term
```

Comment:
```suggestion
    # For VersionType.internal, version is a tuple composed of the sequence number and primary term. For VersionType.none and .create_only, it is None.
```

But check the correctness of this statement.